modules/ui/app/scripts/controllers/DialogModalCtrl.js

```diff
@@ -2,12 +2,12 @@
 
 angular.module('wasabi.controllers')
     .controller('DialogModalCtrl',
-        ['$scope', '$modalInstance', 'options', '$timeout',
-            function ($scope, $modalInstance, options, $timeout) {
+        ['$scope', '$modalInstance', 'options', '$timeout', '$sce',
```

Comment:
Suggest using the expanded version for the variable name "sce" just to make it more explanatory while reading code.